website/content/en/preview/provisioner.md

```diff
@@ -59,6 +65,22 @@ spec:
   provider: {}
 ```
 
+## Node deprovisioning 
+
+If neither of these values are set, Karpenter will *not* delete instances. It is recommended to set the `ttlSecondsAfterEmpty` value, to enable scale down of the cluster. 
+
+### spec.ttlSecondsUntilExpired
+
+Setting a value here enables node expiry. After nodes reach the defined age in seconds, they will be deleted, even if in use. This enables nodes to effectively be periodically "upgraded" by replacing them with newly provisioned instances. This functionality may be called "repaving" or "rehydrating". 
+
+Note that Karpenter does not automatically add skew to this value. If multiple instances are created in a small amount of time, they will expire at very similar times. Consider defining a [pod disruption budget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) to prevent excessive workload disruption. 
+
+This value is in seconds. 
+
+### spec.ttlSecondsAfterEmpty
```

Comment:
Can you move this one above the expiration param?  I think it would flow better since you already mention this param in the heading paragraph for the section.